### DISCUSSION TITLE
Change in Securing Credentials?

### DISCUSSION THREAD

**Discussion by akwala** [floccusaddon/floccus#1580] [General]
I do infrequent manual push up syncs. The last couple of times I wasn't asked for my passphrase when I launched the addon from Firefox's toolbar. What has changed?

**Comment by marcelklehr** [comment]
Hi @akwala 
yes, the feature of securing credentials with a passphrase was removed in v5.x due to migrating to Chrome's manifest v3. The feature was never very secure, though, as it only ever "protected" against an attacker who already has access to your computer, which could still lead to a compromise of your credentials e.g. by the attacker installing a keylogger.
